### PR TITLE
Add Diagrid conductor to integrations

### PR DESCRIPTION
**Pull request by marcduiker**
Signed-off-by: Marc Duiker <marcduiker@users.noreply.github.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add Diagrid Conductor to the Dapr Integrations section.

## Issue reference

#4111

### CODE REVIEW

**Comment by yaron2** [comment]
I suggest we refactor the existing content a little and create a new section titled "Enterprise" (like the Cilium project has) or "Ecosystem" (Like the Istio project has) for this purpose and move some of the stuff under integrations there.

File: daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+linkTitle: "Diagrid Conductor"
+description: "Automate operations, enforce security best practices, improve uptime, and elevate insights across all your Dapr clusters"
+weight: 2000
+---
+
+<br /><img src="/images/diagrid-conductor.png" alt="Diagrid Conductor diagram" width="800">
+
+[Diagrid Conductor](https://www.diagrid.io/conductor) quickly and securely connects to all your Kubernetes clusters running Dapr and Daprized applications, delivering operational excellence, security & reliability, and insights & collaboration.
+
+**Automated Dapr Management**
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md]:
```suggestion
**Automated Dapr management**
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+One-click installation, upgrade and patching of Dapr with selective application update and automated rollback means you’re always up to date.
+
+**Advisor: Automated best-practices**
+
+Be informed and apply production best practices, with continuous checking to prevent misconfigurations, increasing security, reliability and performance.
+
+**Resource Usage Reporting & Tracking**
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md]:
```suggestion
**Resource usage reporting  and tracking**
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+By studying past resource behavior, recommend application resource optimization usage leading to significant cost savings on CPU and memory.
+
+**Application Visualizer**
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md]:
```suggestion
**Application visualizer**
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+<br /><img src="/images/diagrid-conductor.png" alt="Diagrid Conductor diagram" width="800">
+
+[Diagrid Conductor](https://www.diagrid.io/conductor) quickly and securely connects to all your Kubernetes clusters running Dapr and Daprized applications, delivering operational excellence, security & reliability, and insights & collaboration.
+
+**Automated Dapr Management**
+
+One-click installation, upgrade and patching of Dapr with selective application update and automated rollback means you’re always up to date.
+
+**Advisor: Automated best-practices**
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md]:
```suggestion
**Advisor: Discover and automate best practices**
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+type: docs
+title: "Conductor: Enterprise Dapr for Kubernetes"
+linkTitle: "Diagrid Conductor"
+description: "Automate operations, enforce security best practices, improve uptime, and elevate insights across all your Dapr clusters"
+weight: 2000
+---
+
+<br /><img src="/images/diagrid-conductor.png" alt="Diagrid Conductor diagram" width="800">
+
+[Diagrid Conductor](https://www.diagrid.io/conductor) quickly and securely connects to all your Kubernetes clusters running Dapr and Daprized applications, delivering operational excellence, security & reliability, and insights & collaboration.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md]:
```suggestion
[Diagrid Conductor](https://www.diagrid.io/conductor) quickly and securely connects to all your Kubernetes clusters running Dapr and Daprized applications, delivering operational excellence, security & reliability and insights & collaboration.
```

File: daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Conductor: Enterprise Dapr for Kubernetes"
+linkTitle: "Diagrid Conductor"
+description: "Automate operations, enforce security best practices, improve uptime, and elevate insights across all your Dapr clusters"
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/integrations/Diagrid/diagrid-conductor.md]:
```suggestion
description: "Automate operations, enforce security best practices, improve uptime, and elevate insights across your Dapr clusters"
```

**Review by msfussell — APPROVED** [review]
LGTM

**Comment by msfussell** [comment]
@

> I suggest we refactor the existing content a little and create a new section titled "Enterprise" (like the Cilium project has) or "Ecosystem" (Like the Istio project has) for this purpose and move some of the stuff under integrations there.

@yaron2 - I would create an ecosystem heading then instead on integrations. There is this on the website https://dapr.io/enterprise/

**Pull request merged**